### PR TITLE
[MERGE WITH GIT FLOW] Change addresses/text changes on contact page

### PR DESCRIPTION
**Pull request by johnnyporkchops**
Change address on home and data footers and make ext changes to contact page.
Addresses: Hardcoded changes for FEC address #1848
### Contact page:
contact
### Home footer:
home_footer
### Data footer:
data_footer

### CODE REVIEW

File: fec/data/templates/layouts/main.jinja [hunk 1]
```diff
@@ -109,7 +109,7 @@
           </li>
         </ul>
 
-        <p>999 E Street, NW<br>
+        <p>1050 First Street NE<br>
```
Comment (lbeaufort) [inline fec/data/templates/layouts/main.jinja]: Should this have a comma? `1050 First Street, NE`

File: fec/fec/templates/base.html [hunk 2]
```diff
@@ -126,7 +126,7 @@ <h2 style="color: #ffffff">Your browser is outdated</h2>
             </li>
           </ul>
 
-          <p>999 E Street, NW<br>
+          <p>1050 First Street NE<br>
```
Comment (lbeaufort) [inline fec/fec/templates/base.html]: Same as above- should this have a comma before `NE`?

Comment (AmyKort) [reply fec/fec/templates/base.html]: Yes. Thanks!

Comment (AmyKort) [reply fec/fec/templates/base.html]: 1050 First Street, NE

**Pull request merged**